### PR TITLE
Refactored Bquiz UI

### PR DESCRIPTION
**Pull request by NeerajYadu07**
https://user-images.githubusercontent.com/100270646/198830187-79b4f379-112d-47ff-ac6e-94e0ff0716b9.mp4

### CODE REVIEW

**Comment by TreavVasu** [comment]
LGTM review it and notify , if it's al right resolve conflicts and merge

**Comment by NeerajYadu07** [comment]
sir conflicts have been resolved please review it.

File: lib/screens/b_quiz/quiz_repository.dart [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+  }
+}
+
+class APIQuizRepository extends QuizRepository {
+  final String classTag = "APIgetQuizRepository";
+  @override
+  Future<List<Questions>> getAllQuizes() async {
+    final String tag = classTag + "getAllQuizs()";
+    http.Response response;
+    Uri url = Uri.parse("http://43.205.53.122/bquiz/Quizboard/");
```
Comment (TreavVasu) [inline lib/screens/b_quiz/quiz_repository.dart]: change it to base url Use the string constants already present there .Resolve this and notify

Comment (NeerajYadu07) [reply lib/screens/b_quiz/quiz_repository.dart]: @TreavVasu sir issue has been resolved.

**Pull request merged**